src/plugins/borrow-plugins/plugins/aave/AaveBorrowEngineFactory.ts

```diff
@@ -508,6 +507,71 @@ export const makeAaveBorrowEngineFactory = (blueprint: BorrowEngineBlueprint) =>
         }
 
         return composeApprovableActions(...actions)
+      },
+
+      // Utilities
+      async getAprQuote(tokenId?: string): Promise<number> {
+        const token = getToken(tokenId)
+        const tokenAddress = getTokenAddress(token)
+
+        if (tokenAddress == null) {
+          throw new Error(`getAprQuote: AAVE requires a contract address, but tokenId '${tokenId ?? ''}' has none`)
+        }
+
+        const { variableApr } = await aaveNetwork.getReserveTokenAprRates(tokenAddress)
+
+        return variableApr
+      },
+      async getLoanAssetETHValue(loanAsset: BorrowCollateral | BorrowDebt): Promise<string> {
```

Comment:
We should make `getLoanAssetETHValue` private because it's an internal function.